presto-main/src/test/java/com/facebook/presto/execution/TestResetSessionTask.java

```diff
@@ -39,21 +48,39 @@
 
 public class TestResetSessionTask
 {
+    private static final String CATALOG_NAME = "catalog";
     private final ExecutorService executor = newCachedThreadPool(daemonThreadsNamed("stage-executor-%s"));
-    private final MetadataManager metadata = MetadataManager.createTestMetadataManager();
+    private final TransactionManager transactionManager;
+    private final AccessControl accessControl;
+    private final MetadataManager metadata;
 
     public TestResetSessionTask()
     {
+        CatalogManager catalogManager = new CatalogManager();
+        transactionManager = createTestTransactionManager(catalogManager);
+        accessControl = new AllowAllAccessControl();
+
+        metadata = new MetadataManager(
```

Comment:
This is repeated. It needs a helper method somewhere. Maybe in `MetadataManager`?